marklogic-data-hub-central/ui/src/components/header/system-info.test.tsx

```diff
@@ -47,8 +49,36 @@ describe('Update data load settings component', () => {
         expect(getByText('MarkLogic version:')).toBeInTheDocument();
         expect(getByText(data.environment.marklogicVersion)).toBeInTheDocument();
         expect(getByText('Download Configuration Files')).toBeInTheDocument();
-        expect(getByText('Clear All User Data')).toBeInTheDocument();
+        expect(getByTestId('clearData')).toBeInTheDocument();
         expect(getByText('Download')).toBeDisabled();
+        expect(getByText('Clear')).toBeDisabled();
+    });
+
+    test('verify project info display, user with "Clear" button enabled', async () => {
+        const authorityService = new AuthoritiesService();
+        authorityService.setAuthorities(['clearUserData']);
+        const { getByText,getByTestId } = render(<Router><AuthoritiesContext.Provider value={authorityService}>
+            <SystemInfo {...data.environment}
+                        systemInfoVisible={true}
+                        setSystemInfoVisible={jest.fn()}
+            />
+        </AuthoritiesContext.Provider></Router>);
+        expect(getByText(data.environment.serviceName)).toBeInTheDocument();
+        expect(getByText('Data Hub version:')).toBeInTheDocument();
+        expect(getByText(data.environment.dataHubVersion)).toBeInTheDocument();
+        expect(getByText('MarkLogic version:')).toBeInTheDocument();
+        expect(getByText(data.environment.marklogicVersion)).toBeInTheDocument();
+        expect(getByText('Download Configuration Files')).toBeInTheDocument();
+        expect(getByTestId('clearData')).toBeInTheDocument();
+        expect(getByText('Download a zip file containing flow definitions, step definitions and other user artifacts created or modified by Hub Central.')).toBeInTheDocument();
+        expect(getByText('Delete all user data in STAGING, FINAL, and JOBS databases.')).toBeInTheDocument();
+        expect(getByText('Download')).toBeDisabled();
+        expect(getByText('Clear')).toBeEnabled();
+        let clearBtn = getByText('Clear');
+        await wait (()=> {
+            fireEvent.submit(clearBtn);
+        });
+        expect(getByTestId('alertTrue')).toBeInTheDocument();
```

Comment:
Clear data alert has display:'none', so the alert might show up in the DOM passing this assertion even when alert message doesnt appear.
toBeVisible() may be better suited in this situation. To verify using the text which has inline bold tags, refer getSubElements() in Run.test.tsx